engines/python/setup/djl_python/properties_manager/lmi_dist_rb_properties.py

```diff
@@ -63,6 +63,8 @@ class LmiDistRbProperties(Properties):
     cpu_offload_gb_per_gpu: Optional[int] = 0
     enable_prefix_caching: Optional[bool] = False
     disable_sliding_window: Optional[bool] = False
+    limit_mm_per_prompt: Optional[Mapping[str, int]] = None
+    mm_processor_kwargs: Optional[dict] = None
```

Comment:
how are we going to handle these kwargs? if a user passes in something like 
`option.mm_processor_kwargs={"num_crops": 4}`, is that translated correctly?

Reply:
I didn't handle mm_processor_kwargs because it's added in vllm 0.6.2. So I only add it in the properties, but not handle it.